inc/AI.hpp

```diff
@@ -0,0 +1,108 @@
+/*
+** EPITECH PROJECT, 2024
+** *
+** File description:
+** AI
+*/
+
+#ifndef AI_HPP
+    #define AI_HPP
+
+    #include <iostream>
+    #include "Stone.hpp"
+    #include "Board.hpp"
+
+namespace Gomoku {
+    /**
+     * @brief Class for the AI
+     *
+     * This class contains the AI of the game
+     */
+    class AI {
+        public:
+            /**
+             * @brief Construct a new AI object
+             *
+             */
+            AI(Board _board) : _board(_board)
```

Comment:
I dont think we need this constructor, we will always call the default ctor that creates the board itself